ParticleObjects/particleDungeon_class.f90

```diff
@@ -541,10 +610,13 @@ subroutine printToFile(self, name)
 
     ! Print out each particle co-ordinate
     do i = 1, self % pop
-      write(10,'(8A)') numToChar(self % prisoners(i) % r), &
-                       numToChar(self % prisoners(i) % dir), &
-                       numToChar(self % prisoners(i) % E), &
-                       numToChar(self % prisoners(i) % G)
+      ! write(10,'(8A)') numToChar(self % prisoners(i) % r), &
```

Comment:
Should we remove the commented bit here?

Reply:
Has been removed now